frontend/packages/akr/public/i18n/sv-SE/accessibility.json

```
@@ -1 +1,83 @@
+              "title": "Möjlig att uppfatta: Tabellrubrik saknas"
+            },
+            {
+              "description": "Språkvalskomponenten har ingen traditionell ledtext/etikett. Det finns emellertid en Aria-label.",
+              "claim": "2.4.6 Rubriker och ledtexter/etiketter",
+              "title": "Hanterbar: Ledtext/etikett saknas i språkvalskomponenten"
+            },
+            {
+              "description": "Komponenten ”Visa på sida” i tabellen har ingen traditionell ledtext/etikett. Det finns emellertid en Aria-label.",
+              "claim": "2.4.6 Rubriker och ledtexter/etiketter",
+              "title": "Hanterbar: Ledtext/etikett saknas i komponenten ”Visa på sida”"
+            }
+          ]
+        },
+        "contactAdministrativeAgency": {
+          "links": {
+            "email": {
+              "link": "saavutettavuus@avi.fi",
+              "title": "saavutettavuus@avi.fi"
+            },
+            "website": {
+              "ariaLabel": "Enheten för tillgänglighetstillsyn webbplats, öppnas i ett nytt fönster",
+              "link": "https://www.tillganglighetskrav.fi/",
+              "title": "Enheten för tillgänglighetstillsyn webbplats"
+            }
+          },
+          "name": "Regionförvaltningsverket i Södra Finland",
+          "phone": "telefonnummer växel 0295 016 000",
+          "title": "Tillsynsmyndighetens kontaktuppgifter",
+          "unit": "Enheten för tillgänglighetstillsyn"
+        },
+        "feedback": {
+          "description": "Per e-post",
+          "subtitle": "Märkte du ett fel i våra digitala tjänster? Berätta om det för oss och vi gör vårt bästa för att åtgärda bristen.",
+          "title": "Respons och kontaktuppgifter"
+        },
+        "furtherImprove": {
+          "description": "Vi har förbundit oss till att förbättra tillgängligheten i digitala tjänster.",
+          "extraDescription": "Märkte du en brist i tillgängligheten? Berätta om det för oss och vi gör vårt bästa för att åtgärda bristen.",
+          "subtitle": "Vi erbjuder stöd åt de användare för vilka digitala tjänster inte är tillgängliga",
+          "title": "Vi arbetar ständigt för att förbättra tillgängligheten"
+        },
+        "nonAccessible": {
+          "description": "Webbplatsen är ännu inte till alla delar förenlig med kraven",
+          "title": "Icke tillgängligt innehåll"
+        },
+        "status": {
+          "description": "Uppfyller tillgänglighetskraven delvis",
+          "title": "Status för tillgänglighet i den digitala tjänsten"
+        }
+      },
+      "heading": {
+        "description": "Detta tillgänglighetsutlåtande gäller tjänsten akr.opintopolku.fi och har utarbetats/uppdaterats 04.10.2023. Tjänsten omfattas av lagen om tillhandahållande av digitala tjänster, där det förutsätts att offentliga nättjänster ska vara tillgängliga. Tjänstens tillgänglighet har bedömts av en utomstående expertorganisation.",
```

Muissa kieliversioissa välilyönnit kauttaviivan molemmin puolin, voisi varmaan olla tässäkin:
```suggestion
        "description": "Detta tillgänglighetsutlåtande gäller tjänsten akr.opintopolku.fi och har utarbetats / uppdaterats 04.10.2023. Tjänsten omfattas av lagen om tillhandahållande av digitala tjänster, där det förutsätts att offentliga nättjänster ska vara tillgängliga. Tjänstens tillgänglighet har bedömts av en utomstående expertorganisation.",
```